cli.js

```diff
@@ -17,6 +17,7 @@ const cli = meow(`
 	Examples
 	  $ replace-in-files --string='horse' --regex='unicorn|rainbow' --replacement='🦄' foo.md
 	  $ replace-in-files --regex='v\\d+\\.\\d+\\.\\d+' --replacement=v$npm_package_version foo.css
+	  $ replace-in-files --string='blob' --replacement='blog' 'some/**/[gb]lob/*'
```

Comment:
Would be good to show off that it supports multiple patterns and also negation.
```suggestion
	  $ replace-in-files --string='blob' --replacement='blog' 'some/**/[gb]lob/*' '!some/glob/foo'
```